articles/virtual-network/ip-services/routing-preference-unmetered.md

```diff
@@ -25,7 +25,7 @@ Your network traffic egressing from origin in Azure destined to CDN provider ben
 
 ## Configuring Routing Preference Unmetered
 
-To take advantage of routing preference unmetered, your CDN providers need to be part of this program. If your CDN provider isn't part of the program, contact your CDN provider.
+To take advantage of routing preference unmetered, your CDN provider needs to be part of this program. Check with your CDN provider on which CDN services are supported by routing preference unmetered. If your CDN provider isn't part of the program, contact your CDN provider. For a list of Azure services that are supported by routing preference unmetered see [https://learn.microsoft.com/en-US/azure/virtual-network/ip-services/routing-preference-overview#supported-services].
```

Comment:
@callachun The link you reference in the update does not go to a list of services specifically called out as supported by **unmetered**. Only **Microsoft global network** and **Internet** are in doc. Does a list need to be added to the doc or do you have another resource?

Also, the links needs to be re-written. Please use the following:

*For a list of Azure services that supported by routing preference unmetered see [Routing preferences in Azure](./routing-preference-overview.md#supported-services)